src/common/dto/user/responses/user.response.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Expose } from 'class-transformer';
+import { IsString } from 'class-validator';
+import { UserDto } from '../user.dto';
+
+export class UserResponse extends UserDto {
+	@ApiProperty({ example: 1 })
+	@Expose()
+	@IsString()
+	id?: number;
```

Comment:
Incorrect swagger description for this field! TypeScript says this field is optional but swagger says it is required.